src/Grpc.Net.ClientFactory/Grpc.Net.ClientFactory.csproj

```diff
@@ -11,6 +11,13 @@
     <VersionPrefix>$(GrpcDotnetVersion)</VersionPrefix>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
File an issue to remove this workaround later. Also, can we move this to a Directory.Build.target file?